Use Cases Onboarding (#6800)
segmentio/segment-docs: src/getting-started/use-cases/setup.md

---
title: Use Cases Setup
hidden: true
---

Use Cases help you onboard quickly and efficiently to Segment by guiding you through specific steps tailored to your business needs.

This page walks you through the steps to set up a use case in your Segment instance.

> info "Permissions"

> > To implement a use case, you'll need to be a workspace owner for your Segment account.

Reviewer: ```suggestion
> To implement a use case, you'll need to be a Workspace Owner for your Segment account. See the [Roles](/docs/segment-app/iam/roles/) documentation for more information. 
```

## Use case setup overview

From a high level, setting Segment up with a use case takes place in four stages:

> 1. **Pick your business goal**. What do you want to achieve?  Choose from goals like optimizing advertising, personalizing first conversions, and increasing customer retention.

Reviewer: I don't know if you necessarily need this level of detail in the setup guide, as folks presumably have already selected a use case, but might be helpful to note that these are pre-defined

```suggestion
1. **Pick your business goal**. What do you want to achieve?  Choose from 4 common business goals like optimizing advertising, personalizing first conversions, boosting retention, and increasing customer retention. 
```

Author: No, this is really good advice.

Segment strongly recommends beginning your setup in the dev environment. This allows for thorough testing and debugging of your configuration. Once you’re confident in your dev setup, Segment will guide you on how to apply these configurations to your live production sources.

### Step 3: Review suggested events

> warning "Changing your use case"
> Once you've reviewed the suggested events for a use case, you won't be able to change the use case. If you want to see a full breakdown of each use case before commiting to one, click **Change use case** to begin the use case flow again. You can also view the [Use Cases Reference guide](/docs/getting-started/use-cases/reference/) to see what Segment recommends for each use case.

On the **Setup checklist** page, you'll see the full checklist for the use case you've chosen. This checklist applies to all use cases, though the suggested events, sources, and destinations differ between use cases.

1. In the **Review suggested events** list item, click **Review**. 
2. Segment shows you the recommended events and properties typically tracked for your use case. 
3. Set up event tracking based on the events and properties Segment shows.

> This table shows Segment's recommended events and properties for the Personaliz winback use case:

Reviewer: ```suggestion
This table shows Segment's recommended events and properties for the Personalize winback use case:
```

| Order Completed | `num_items`, `order_id`, `checkout_id`, `total`, `revenue`, `shipping`, `tax`, `affiliation`, `products` |

Make sure that you're tracking these events to get the most of the Personalize winback campaign. For more information on event tracking, see [Data Collection Best Practices](/docs/protocols/tracking-plan/best-practices/).

### Step 4: Connect dev sources 

You're now ready to connect sources to your dev environment. 

1. In the **Connect dev sources** step, Segment shows you the recommended sources you should connect. For Personalized winback, these include Website, Mobile, and Reverse ETL.
2. Review the recommended sources, then click **Connect**.
3. Segment takes you to the **Add a source** setup. Choose the source(s) you want to add, then click **Next**.
4. Name your source, then click **Create source**. 
5. Carry out the source-specific steps, then click **Next**.
6. Test your connection and troubleshoot it, if necessary. Click **Done**.
7. (Optional:) Click **Connect More** and repeat steps 2 through 6 to add more sources.

> info "Adding a warehouse as a souce"

> > If you connect a warehouse as a source, Segment automatically creates a Profiles destination that shows up in the **Connect your data** tab. Segment requires this destination to create profiles from your warehouse. 

Reviewer: Deleting a pre-made destination was a big deal in Unified Profiles: not sure if it is in this case too. Disregard if it's nbd

```suggestion
> If you connect a warehouse as a source, Segment automatically creates a Profiles destination that shows up in the **Connect your data** tab. Do not delete this destination, as Segment requires this destination to create profiles from your warehouse. 
```

> info "Cloud object sources"
> If you connect a cloud object source, you'll need to create a warehouse to sync profiles into Segment. For more information, see [Cloud Sources](/docs/connections/sources/about-cloud-sources/).

### Step 5: Connect dev destinations 

With sources connected, you can now connect destinations to your dev environment.

> 1. Under the **Connect dev destinations** step, Segment shows you the recommended sources you should connect. For Personalized winback, these include Reverse ETL, Personaliztion, and Analytics.

Reviewer: ```suggestion
1. Under the **Connect dev destinations** step, Segment shows you the recommended sources you should connect. For Personalize winback, these include Reverse ETL, Personalization, and Analytics.
```

Author: Ugh, thanks for saving me from myself, as usual.

3. Segment takes you to the **Choose a Destination** setup. Choose the destination(s) you want to add, then click **Next**.
4. Name your destination, then click **Create Destination**.
4. Choose a source to connect to the destination, then click **Next**. 
5. Carry out the destination-specific steps, then click **Done**.
7. (Optional:) Click **Connect More** and repeat steps 2 through 6 to add more destinations.

### Step 6: Publish your setup to a prod environment

Until this point, you've set up event tracking and connected sources and destinations to a development environment. 

After you've confirmed that data is flowing from your sources into your destinations as expected, you're ready to publish your setup to a production environment.

1. On the Setup checklist page, click the **Prod environment** tab.
2. On the **Connect 1 prod source** radio button, click **Connect**. 
3. Segment shows you the sources you previously connected in your dev environment. Click the source you want to connect to prod, then click **Continue**.
4. Carry out any additional steps in the Add a Source page, click **Create Source**, then click **Next**. Segment returns you to the Prod environment tab.

> 5.  Publish the events set up in your dev environment sources to production. Check the debugger to verify that data is flowing into Segment correctly, then click **Mark as complete**.

Reviewer: ```suggestion
5. Publish the events set up in your dev environment sources to production. Check the debugger to verify that data is flowing into Segment correctly, then click **Mark as complete**.
```
